src/routes/instanceManager/InstanceManagerList.js

```diff
@@ -13,11 +13,9 @@ function list({ loading, dataSource, height, showRefCountVolumeModal }) {
       sorter: (a, b) => sortTable(a, b, 'name'),
       render: (text, record) => {
         return (
-          <div>
-            <Button type="link" onClick={() => showRefCountVolumeModal(record.volume)}>
-              {text}
-            </Button>
-          </div>
+          <a className="ant-btn-link" onClick={() => showRefCountVolumeModal(record.volume)}>
+            {text}
+          </a>
```

Comment:
Can you explain why this change can solve the wrapping issue?

Reply:
The property "white-space: nowrap;" on the class ".ant-btn" of antd `Button` prevents text from wrapping according to its width.

Reply:
The main reason why Button does not wrap is the CSS property "white-space: nowrap;", which is added to the Button component by default in antd. To fix this issue while keeping the style unchanged, all we need to do is replace the Button component with an "a" link and add the ".ant-btn-link" class name.

Reply:
:+1: